### PR TITLE
Bug/zoom box

### PR DESCRIPTION
**Pull request by dvantetering**
- ZoomBox scales with screen width
- Most Recent bug found and fixed

### CODE REVIEW

File: src/main/java/application/controllers/GraphController.java [hunk 1]
```diff
@@ -287,6 +302,16 @@ public void update(ArrayList<String> ref, int depth) {
             for (Edge e : graph.getModel().getAddedEdges()) {
                 checkEdgeLength(e, maxEdgeLength, maxEdgeLengthLong);
             }
+
+            Set<Node> nodes = this.getRoot().lookupAll(".scroll-bar");
+            nodes.stream().filter(node -> node instanceof ScrollBar).forEach(node -> {
+                ScrollBar sb = (ScrollBar) node;
+                sb.valueProperty().addListener((ObservableValue<? extends Number> ov,
+                                                Number old, Number newval) -> {
```
Comment (Warnars) [inline src/main/java/application/controllers/GraphController.java]: Can't this be placed on one line?

**Comment by tieswesterborg** [comment]
No remarks. Works as expected

File: src/main/java/application/fxobjects/ZoomBox.java [hunk 2]
```diff
@@ -114,25 +117,30 @@ public void replaceZoomBox(double[] places) {
...
+            shown = 1.0;
+        }
+
+        if (rightOffset > 1.0) {
+            rightOffset = 1.0;
+        }
+
+        double right = rectX + rightOffset * zoomBoxWidth;
         double newWidth = shown * zoomBoxWidth;
+        double maxX = rectX + zoomBoxWidth -  newWidth;
```
Comment (ghost) [inline src/main/java/application/fxobjects/ZoomBox.java]: Contains space char too much

File: src/main/java/application/fxobjects/layout/GraphLayout.java [hunk 3]
```diff
@@ -241,6 +246,11 @@ public double getMaxWidth() {
     }
 
     /**
+     * Get the max height
+     * @return the max height
+     */
+    public double getMaxHeight() { return maxHeight; }
```
Comment (ghost) [inline src/main/java/application/fxobjects/layout/GraphLayout.java:252]: Method body should not be placed on the same line as the method definition.

Comment (ric-jon) [inline src/main/java/application/fxobjects/layout/GraphLayout.java:252]: Allowed for trivially simple methods like setters & getters isn't it?

Comment (dvantetering) [inline src/main/java/application/fxobjects/layout/GraphLayout.java:252]: Yeah I thought so

Comment (tieswesterborg) [inline src/main/java/application/fxobjects/layout/GraphLayout.java:252]: JavaDoc for getters and setters are not mandatory.

**Pull request merged**